module.json

```diff
@@ -36,11 +36,11 @@
     "minar": "^1.0.0",
     "core-util": "^1.0.0",
     "compiler-polyfill": "^1.1.0",
-    "greentea-client": "^0.1.4"
+    "greentea-client": "~0.1.4"
```

Comment:
If you list `greentea-client` as a regular dependency, it'll always be installed, so there's no need to also list it as a test dependency. I _think_ it should be a test dependency, but I haven't check if this is correct.